nixos/tests/mmdbctl.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, pkgs, ... }:
+{
+
+  name = "mmdbctl";
+
+  meta = {
+    maintainers = with lib.maintainers; [ moraxyc ];
+  };
+
+  nodes.machine =
+    { pkgs, ... }:
+    {
+      environment.systemPackages = [ pkgs.mmdbctl ];
+    };
+
+  testScript = ''
+    # DBIP-Country-Lite
+    machine.succeed("mmdbctl verify ${pkgs.dbip-country-lite}/share/dbip/dbip-country-lite.mmdb | grep valid")
```

Comment:
Why is this a NixOS test? Is it necessary to bind to some ports or dbus?

Reply:
Yes, I shouldn't use nixosTests. I'm writing `passthru.tests` in the package.